packages/types/src/localization.ts

```diff
@@ -761,14 +762,35 @@ type _LocalizationResource = {
         headerTitle__subscriptions: LocalizationValue;
         headerTitle__statements: LocalizationValue;
       };
+      statementsSection: {
+        empty: LocalizationValue;
+        itemCaption__paidForPlan: LocalizationValue;
+        itemCaption__proratedCredit: LocalizationValue;
+        itemCaption__subscribedAndPaidForPlan: LocalizationValue;
+        notFound: LocalizationValue;
+        tableHeaders__date: LocalizationValue;
+        tableHeaders__amount: LocalizationValue;
```

Comment:
Align with existing `tableHeader` from orgs tables
```suggestion
        tableHeader__date: LocalizationValue;
        tableHeader__amount: LocalizationValue;
```